src/middleware/parser.js

```diff
@@ -75,7 +75,11 @@ const parseOutgoingBody = (ctx, outputFormat) => {
 }
 
 const parseIncomingBody = async (ctx, inputFormat) => {
-  // parse incoming body
+  ctx.state.allData.query = ctx.request.query
+
+  // parse incoming body if request is of type post only
+  if (ctx.request.method === 'GET') return
```

Comment:
We might need to include the `DELETE` method as well

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/79/commits/2665ac33eb80572fdb853816f7b5d48de33601ef